e2e/rook-test-framework/clients/k8s_rook_block.go

```diff
@@ -0,0 +1,138 @@
+package clients
+
+import (
+	"errors"
+	"github.com/rook/rook/e2e/rook-test-framework/contracts"
+)
+
+type k8sRookBlock struct {
+	transportClient contracts.ITransportClient
+}
+
+var (
+	listCmd              = []string{"rook", "block", "ls"}
+	wideDataToBlockPod   = []string{"sh", "-c", "WRITE_DATA_CMD"}
+	readDataFromBlockPod = []string{"cat", "READ_DATA_CMD"}
+)
+
+// Constructor to create k8sRookBlock - client to perform rook Block operations on k8s
+func CreateK8sRookBlock(client contracts.ITransportClient) *k8sRookBlock {
+	return &k8sRookBlock{transportClient: client}
+}
+
+// Function to create a Block using Rook
+// Input paramaters -
+//name - path to a yaml file that creates a pvc in k8s - yaml should describe name and size of pvc being created
+//size - not user for k8s implementation since its descried on the pvc yaml definition
+//Output - k8s create pvc operation output and/or error
+func (rb *k8sRookBlock) Block_Create(name string, size int) (string, error) {
```

Comment:
when creating the block we need to specify the size, it's part of the create block API and block_image model